src/Core/AcmeClient.php

```diff
@@ -155,7 +155,11 @@ public function finalizeOrder(CertificateOrder $order, CertificateRequest $csr,
         if (\in_array($response['status'], ['pending', 'processing', 'ready'])) {
             $humanText = ['-----BEGIN CERTIFICATE REQUEST-----', '-----END CERTIFICATE REQUEST-----'];
 
-            $csrContent = $this->csrSigner->signCertificateRequest($csr);
+            $csrContent = $csr->getCsrPem();
```

Comment:
You can inline this:

```
$csrContent = $csr->getCsrPem() ?: $this->csrSigner->signCertificateRequest($csr);
```

Reply:
Honestly, I'm old-scholl C programmer (Linux kernel style), don't like inlines like this.